src/main/java/ca/corefacility/bioinformatics/irida/ria/web/search/dto/SearchItem.java

```diff
@@ -0,0 +1,36 @@
+package ca.corefacility.bioinformatics.irida.ria.web.search.dto;
+
+import ca.corefacility.bioinformatics.irida.ria.web.models.tables.AntTableItem;
+
```

Comment:
Add class comment

Reply:
Updated in 900cb3df4f1ab894644e1533e69eb2a843fd7d6a